src/model/extract-nav.ts

```diff
@@ -119,3 +119,14 @@ export const extractNAV = (data: any): NavType => {
 		readerRevenueLinks: buildRRLinkModel(data),
 	};
 };
+
+export const forBrowser = (nav: NavType): BrowserNavType => {
+	return {
+		topLevelPillars: nav.pillars.map((p) => ({
+			...p,
```

Comment:
We are mapping over `pillars` in a two-line function literal so I'm not convinced readability is harmed here. (But also, I think there is a risk of bike-shedding here/doesn't matter either way.)

A good rule of thumb is, the further from the point of definition a name will be used, the more descriptive it needs to be. This is why it is okay to use `i` (or equivalent) as index or counter in a loop for example, but function names need to be clearer.

Reply:
Ha, yes, we're defo hanging around the bike sheds with things like this. I will point out though that on another separate PR I got stuck when I came across an `i`. I was _guessing_ that it was index but the context wasn't apparent and searching for 'i' didn't go so well!

The reason this happened was because the code had _grown_ and expanded over time and the definition and usage and spread apart. So I completely agree that where the point of definition is close it's a moot point, the problem with that is things will change so we whoudl write defensive code that will scale well.

Reply:
Defo a nitpick yeah, so feel free to resolve the conversation and move on but one of the main reasons I don't like single letter params is because I have always found it is one of the reasons idiomatic scala has a huge cognitive load for me.